extensions/amp-install-serviceworker/amp-install-serviceworker.md

```diff
@@ -16,9 +16,45 @@ limitations under the License.
 
 ### <a name="amp-install-serviceworker"></a> `amp-install-serviceworker`
 
-The `amp-install-serviceworker` component allows installing a [ServiceWorker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/) for the current page.
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Installs a [ServiceWorker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/) for the current page.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-install-serviceworker" src="https://cdn.ampproject.org/v0/amp-install-serviceworker-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>None</td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-install-serviceworker` tag
+(see also `amp-install-serviceworker` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
```

Comment:
Your docs made me notice that validation should require `src` and a NODISPLAY `layout`. I'm making that change now. You may want to document those here too.